fastestimator/op/tensorop/loss/super_loss.py

```diff
@@ -80,29 +78,7 @@ def __init__(self,
 
     def build(self, framework: str, device: Optional[torch.device] = None) -> None:
```

Comment:
We can ignore the framework parameter and throw depritiated warning and not condition on framework